examples/cuda/group_stride/Makefile

```diff
@@ -0,0 +1,133 @@
+# Copyright (c) 2021, University of Washington All rights reserved.
```

Comment:
were we ever going to do that thing where we generate these makefiles?

Reply:
Maybe